describe/describePrivate.ts

```diff
@@ -0,0 +1,5 @@
+export type PrivateDescription = string;
+
+export function describePrivate(element: Element): PrivateDescription {
```

Comment:
Since these are not going to be exported externally but just used internally, I think it would be fine to start a non-constructor function with a capital letter (going against convention) and just name the describe functions by the tag names they describe in order to avoid a repetitive descriptors map.

Reply:
Why not just have it be a lower case like `private`, `text`, `scl` ... ?

Reply:
I do like the idea that I could do a case sensitive search on a tag name if we went against convention.

Reply:
It feels strange to me I must say. With the comment from @danyill  though I does it then for me as well. Let's use upper case here.